components/AddModal/AddModal.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+
+import Image from "next/image";
+
+import { allowScroll, preventScroll } from "lib/modal";
+
+import styles from "./AddModal.module.css";
+import AddPortalWrapper from "./AddPortalWrapper";
+import FolderItem from "./FolderItem";
+
+interface IAddModal {
```

Comment:
컴포넌트의 props을 나타낼때는 보통 뒤에 props를 붙이는게 관례입니디ㅏ.
```suggestion
interface IAddModalProps {
```